docs/content/guides/7.multistore/3.patterns/5.subpath/1.subpath.md

```diff
@@ -0,0 +1,819 @@
+} as Config;
+```
+
+Then use these variants (`electronics:`, `apparel:`, etc.) in your components:
+
+```vue
+// apps/storefront-unified-nuxt/components/ui/NavbarTop.vue
+<template>
+  <header
+    :class="[
+      'sticky top-0 z-40 flex h-14',
+      'bg-primary-800',
+      'electronics:!bg-primary-600',
+      'apparel:!bg-primary-400',
+      'md:-top-5 md:h-20 md:pt-2.5',
+    ]"
+  >
+    <slot />
+  </header>
+</template>
+```
+::
+
+::tip Using CSS Variables with Tailwind
+The CSS variables we defined can be used with Tailwind's color system. For example, `bg-primary-500` will use the store-specific primary color defined in the CSS variables. For more information, see the [Tailwind CSS documentation](https://v3.tailwindcss.com/docs/customizing-colors#using-css-variables).
+::
+
+::tip Understanding Generated CSS
+When using store-specific variants like `electronics:!bg-primary-600`, Tailwind generates scoped CSS rules. For example:
+
+```css
+.electronics .electronics\:\!bg-primary-600 {
+    --tw-bg-opacity: 1 !important;
+    background-color: rgb(46 106 230 / 1 !important) !important;
+}
+```
+
+This ensures the styles only apply when two conditions are met: the parent element must have the `.electronics` class and the target element must have the `electronics:!bg-primary-600` class.
+::
+
+## Conditional Components
+
+There are two main approaches to handling store-specific components:
+
+### Approach 1: Different Versions of the Same Component
+
+This approach is useful when you need different variations of a component based on the store, such as announcement bars or featured sections with store-specific content.
+
+::tabs{:titles='["Next.js", "Nuxt"]'}
+
+#tab-1
+```tsx
+// apps/storefront-unified-nextjs/components/announcement-bar.tsx
+import { ApparelAnnouncementBar } from '@/components/store-apparel/announcement-bar';
+import { ElectronicsAnnouncementBar } from '@/components/store-electronics/announcement-bar';
+
+interface AnnouncementBarProps {
+  configId: string;
+}
+
+function AnnouncementBar({ configId }: AnnouncementBarProps) {
+  const StoreComponent = {
+    apparel: ApparelAnnouncementBar,
+    electronics: ElectronicsAnnouncementBar,
+  }[configId];
+
+  return StoreComponent ? <StoreComponent /> : null;
+}
+
+// Usage in a page or layout:
+export default function Layout({ children, params: { configId } }: LayoutProps) {
+  return (
+    <div className={configId}>
+      <AnnouncementBar configId={configId} />
+      {children}
+    </div>
+  );
+}
+```
+
+#tab-2
+```vue
+<!-- apps/storefront-unified-nuxt/components/AnnouncementBar.vue -->
+<template>
+  <component :is="StoreComponent" v-if="StoreComponent" />
+</template>
+
+<script setup lang="ts">
+const props = defineProps<{
+  configId: string;
+}>();
+
+const AnnouncementBarElectronics = resolveComponent('StoreElectronicsAnnouncementBar');
+const AnnouncementBarApparel = resolveComponent('StoreApparelAnnouncementBar');
+const AnnouncementBar = computed(
```

Comment:
suggestion: create a wrapper component to render dynamic component rendering based on `configId`. 

```ts
<StoreComponentRenderer
  components= {
    electronics: ElectronicsAnnouncementBar,
    apparel: ApparelAnnouncementBar
  }
/>
```

It could also be used in this scenario:
```
      {configId === 'electronics' && (
        <ElectronicsPromo />
      )}
```

like this:

```ts
<StoreComponentRenderer
  components= {
    electronics: ElectronicsPromo,
  }
/>
```